config/runtimes/kserve-mlserver.yaml

```diff
@@ -3,6 +3,8 @@ kind: ClusterServingRuntime
 metadata:
   name: kserve-mlserver
 spec:
+  annotations:
+    prometheus.kserve.io/port: '8080'
```

Comment:
good catch, I think I lumped it with the python ones. I think its 8082, checking.

Reply:
mlserver docs seems to suggest its using port 8082 https://mlserver.readthedocs.io/en/latest/reference/settings.html

but I tested it and 8082 doesn't emit prometheus metrics, but 8080/metrics does.

Reply:
tested using https://github.com/kserve/kserve/tree/master/docs/samples/v1beta1/xgboost and confirmed mlserver was used because it set 8082 as the env var

Reply:
From mlserver maintainer, the longer version is that 1.0.0 exposed the metrics endpoint in the same HTTP server as the REST inference endpoints. But on 1.1.0 we decided to move that to a separate server (which runs on port 8082 by default).

Reply:
added a comment in the runtime YAML about this.